nsqadmin/options.go

```diff
@@ -43,5 +45,6 @@ func NewOptions() *Options {
 		StatsdInterval:           60 * time.Second,
 		HTTPClientConnectTimeout: 2 * time.Second,
 		HTTPClientRequestTimeout: 5 * time.Second,
+		AllowConfigFromCIDR:      "127.0.0.1/8",
```

Comment:
This default is technically a breaking change, but probably the _right thing to do_.